content/upgrades/index.md

```diff
@@ -0,0 +1,436 @@
+    * Amazon EKS Add-ons may only be upgraded one minor version at a time. 
+
+[Learn more about what components are available as EKS Add-ons, and how to get started.](https://docs.aws.amazon.com/eks/latest/userguide/eks-add-ons.html)
+
+[Learn how to supply a custom configuration to an EKS Add-on.](https://aws.amazon.com/blogs/containers/amazon-eks-add-ons-advanced-configuration/)
+
+## Identify and remediate removed API usage before upgrading the control plane
+
+You should identify API usage of removed APIs before upgrading your EKS control plane. To do that we recommend using tools that can check a running cluster or static, rendered Kubernetes manifest files. 
+
+Running the check against static manifest files is generally more accurate. If run against live clusters, these tools may return false positives. 
+
+A deprecated Kubernetes API does not mean the API has been removed. You should check the [Kubernetes Deprecation Policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/) to understand API removal affects your workloads.
+
+### Kube-no-trouble
+
+[Kube-no-trouble](https://github.com/doitintl/kube-no-trouble) is an open source command line utility with the command `kubent`. When you run `kubent` without any arguments it will use your current KubeConfig context and scan the cluster and print a report with what APIs will be deprecated and removed. 
+
+```
+kubent
+
+4:17PM INF >>> Kube No Trouble `kubent` <<<
+4:17PM INF version 0.7.0 (git sha d1bb4e5fd6550b533b2013671aa8419d923ee042)
+4:17PM INF Initializing collectors and retrieving data
+4:17PM INF Target K8s version is 1.24.8-eks-ffeb93d
+4:l INF Retrieved 93 resources from collector name=Cluster
+4:17PM INF Retrieved 16 resources from collector name="Helm v3"
+4:17PM INF Loaded ruleset name=custom.rego.tmpl
+4:17PM INF Loaded ruleset name=deprecated-1-16.rego
+4:17PM INF Loaded ruleset name=deprecated-1-22.rego
+4:17PM INF Loaded ruleset name=deprecated-1-25.rego
+4:17PM INF Loaded ruleset name=deprecated-1-26.rego
+4:17PM INF Loaded ruleset name=deprecated-future.rego
+__________________________________________________________________________________________
+>>> Deprecated APIs removed in 1.25 <<<
+------------------------------------------------------------------------------------------
+KIND                NAMESPACE     NAME             API_VERSION      REPLACE_WITH (SINCE)
+PodSecurityPolicy   <undefined>   eks.privileged   policy/v1beta1   <removed> (1.21.0)
+```
+
+It can also be used to scan static manifest files and helm packages. It is recommended to run `kubent` as part of a continuous integration (CI) process to identify issues before manifests are deployed. Scanning manifests is also more accurate than scanning live clusters. 
+
+Kube-no-trouble provides a sample [Service Account and Role](https://github.com/doitintl/kube-no-trouble/blob/master/docs/k8s-sa-and-role-example.yaml) with the appropriate permissions for scanning the cluster. 
```

Comment:
Why do we need this special callout here?

Reply:
these two tools are commonly used, don't want to pick a winner